CIP-0072/README.md

```diff
@@ -0,0 +1,413 @@
+
+## **Motivation**
+DApps can express a plethora of information. Some of this information could be claims about who the developer is, what is the dApp's associated metadata are, and more. This data lacks standarisation, persistence, and immutability. Data without these features, means that dApp users cannot verify if the dApp's expressed information is consistent across time. The goal of this CIP is to formalise how dApps register their information with a new transaction metadata format that can record the dApp's metadata, ownership, and potentially developer's identity. This formalisation means dApp users can verify if the data expressed by a dApp is consistent with what was registered on-chain.
+
+Also having this formalisation facilitates any actor in the ecosystem to index and query this data, and provide a better user experience when it comes to dApp discovery and usage.
+
+## **Definitions**
+- **anchor** - A hash written on-chain that can be used to verify the integrity (by way of a cryptographic hash) of the data that is found off-chain.
+- **dApp** - A decentralised application that is described by the combination of metadata, certificate and a set of used scripts.
+
+## **Specification**
+
+### **Developers / Publishers**
+Developers and publishers of dApps can register their dApps by submitting a transaction on-chain that can be indexed and verified by stores, auditors and other actors. 
+
+### **Stores / Auditors**
+Store and auditors should be able to follow the chain and find when a new dApp registration is **anchored** on-chain. They should then perform *integrity* and *trust* validations on the dApp's certificate and metadata. 
+
+#### **Off-chain Location Advertisement**
+Each store and auditor should make public the location of their off-chain sources where they will look for the dApp's metadata based on certificates found on-chain. These can be advertised through their own API or publically available on a website.
+
+Sample off-chain sources could be for example :
+- [CIP-26](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0026) compliant servers
+- IPFS
+
+##### **Targetted Releases**
+Each developer and publisher can choose to where to write metadata based on the information available from known stores & auditors. This gives **developers** and **publishers** the ability to perform targeted releases. (i.e to which stores and auditors)
+
+#### **Suggested Validations**
+
+- **`integrity`**: The dapp's metadata off-chain should match the metadata anchored on-chain.
+- **`trust`**: The dApp's certificate should be signed by a trusted entity. It's up to the store/auditor to decide which entities are trusted and they should maintain and publish their own list of trusted entities. Although this entities might be well known, it's not the responsibility of this CIP to maintain this list.
+
+### **On-chain dApp Registration certificate**
```

Comment:
Is there any use case for attaching an expiration to metadata registrations? such as "after X epochs this metadata is out of date and should be updated" - Although this may introduce problems with updates being missed by chain followers who are waiting for expirations before checking chain.